get_genotype_vcf.py

```diff
@@ -288,12 +295,13 @@ def main(
             mt = mt.checkpoint(dense_checkpoint)
 
         # filter out related samples from vre too
-        # this will get dropped as the vds file will already be clean
         related_ht = hl.read_table(relateds_to_drop_path)
         related_samples = hl.literal(related_ht.s.collect())
         mt = mt.filter_cols(~related_samples.contains(mt['s']))
 
-        # again filter for biallelic SNPs
+        # again filter for QC-pass, biallelic SNPs
+        qc_pass_variants = hl.read_table(qc_pass_variants_path)
+        mt = mt.semi_join_rows(qc_pass_variants)
```

Comment:
I trust you but I don't really know this syntax, why are we not just filtering rows?

Reply:
The MatrixTable's [semi_join_rows](https://hail.is/docs/0.2/hail.MatrixTable.html#hail.MatrixTable.semi_join_rows) is just a more efficient filter that you can leverage when you only need to match on the table keys (i.e. locus and alleles) - so same end result but, hopefully, a bit faster.